package.json

```diff
@@ -388,16 +375,6 @@
         "vscode-debugadapter": "^1.39.1",
         "wavefile": "^11.0.0"
     },
-    "prettier": {
-        "semi": true,
-        "tabWidth": 4,
-        "printWidth": 120,
-        "trailingComma": "es5",
-        "bracketSpacing": true,
-        "arrowParens": "avoid",
-        "endOfLine": "lf",
-        "singleQuote": true
-    },
```

Comment:
Why we remove those prettier config ? Are they moved to somewhere else ?

Reply:
Yes correct, moved it to `.prettierrc` file